README.md

```diff
@@ -13,9 +13,13 @@ You will need a basic understanding of how to use the command line on your compu
 
 You will need Node and npm installed on your computer. Node is a JavaScript runtime used for developer tooling, and npm is the package manager included with Node. If you have a package manager installed for your operating system, setup can be as straightforward as:
 
-* macOS: `brew install node`
-* Windows: `choco install nodejs`
-* Ubuntu: `apt install nodejs npm`
+* macOS: `brew install node@14`
+* Windows: `choco install nodejs --version=14.20.1`
+* Ubuntu: 01.` sudo apt install curl `
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;02. `curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -`
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;03. `sudo apt install nodejs`
```

Comment:
Use indent instead of number and `&nbsp;`

```suggestion
* Ubuntu: 
	* `sudo apt install curl`
	* `curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash -`
	* `sudo apt install nodejs`
```